gradle/libs.versions.toml

```diff
@@ -1,22 +1,19 @@
 [versions]
-compose-compiler = "1.5.5"
-compose-plugin = "1.5.12"
 agp = "8.2.2"
 android-minSdk = "24"
 android-compileSdk = "34"
 android-targetSdk = "34"
-androidx-activityCompose = "1.8.0"
-jetpack-compose-bom = "2024.01.00"
+androidx-activityCompose = "1.8.2"
+jetpack-compose-bom = "2024.02.02"
+compose-plugin = "1.6.0"
 koin = "3.5.3"
-kotlin = "1.9.20"
-ktor = "2.3.8"
-kamel = "0.9.1"
+kotlin = "1.9.22"
```

Comment:
QQ - can we not update to 1.9.23 now?

Reply:
The wizard's projects are also on 1.9.23, I'll keep it in sync for now.